content/installation/setup/auth/Configure-http-proxy.md

```diff
@@ -5,11 +5,14 @@ tags: "installation setup EOP system settings connect https proxy header"
 -->
 
 
-Contrast supports x509 client certificate authentication through a trusted proxy, which authenticates the user and then sends the user's username to Contrast in a HTTP header. To enable trusted HTTPS proxy authentication: 
+Contrast supports client authentication through a trusted proxy, which authenticates the user and then sends the user's username to Contrast in a HTTP header. (This type of authentication is particularly useful for x509 clients.) Users must be configured in Contrast before starting their authentication configuration, and use the same email address as their usernames for both configurations, in order to be granted access to Contrast after authentication. 
 
-* Update the property file by changing the `authentication.mode` in *~/path_to_contrast_installation/conf/auth.properties* to `http_header`. 
-* By default, the HTTP header name is `Contrast-Authentication`. This is configurable in the *auth.properties* file, if you update the value of `auth.http.header.field.name`.
+To enable trusted HTTPS proxy authentication:
 
+* Update the *auth.properties* property file by changing the `authentication.mode` in *~/path_to_contrast_installation/conf/auth.properties* to `http_header`. 
+* By default, the HTTP header name is `Contrast-Authentication`. You can also configure this in the *auth.properties* file by updating the value of `auth.http.header.field.name`.
 * After restarting Contrast, each request must include the HTTP header `Contrast-Authentication: username` to login. 
 
-> **Note:** Trusted HTTPS proxy authentication should only be used if all Contrast nodes are accessible exclusively through a trusted proxy. No nodes should be accessible directly.
\ No newline at end of file
+> **Note:** Trusted HTTPS proxy authentication should only be used if all Contrast nodes are accessible exclusively through a trusted proxy. No nodes should be accessible directly.
```

Comment:
What has changed about this stanza? Is it just the extraneous whitespace added afterwards? Can we just remove this whitespace?